src/registry/radio/preview.tsx

```diff
@@ -42,6 +42,7 @@ const Preview: React.FC<ComponentPreviewProps<RadioComponentSchema>> = ({compone
       label={label}
       tooltip={tooltip}
       required={required}
+      isPreview={true}
```

Comment:
I'll go with `clearable`, so that it is not related to the preview functionality we have in the builder